build/NuSpecs/WindowsAppSDK-Nuget-Native.targets

```diff
@@ -90,6 +90,12 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.Windows.*.winmd">
```

Comment:
Won't this make duplicate entries for the Microsoft.Windows.*.winmds we already added above?

Reply:
I think we specifically want to add:
Microsoft.Windows.AppNotifications.winmd
Microsoft.Windows.Globalization.winmd
Microsoft.Windows.Management.Deployment.winmd

Reply:
@JesseCol As I understand, as long as the `Private` property is not conflicting, it won't make issues. I applied it as wildcard to avoid any unforeseen issues because we were having a wildcard usage in WinUI that was getting applied to all winmd. But we can go with specific winmd as well.

Reply:
Adding @kythant / @Scottj1s -- do you know who might want to review a change like this?

I'm afraid I don't have enough context to know if this change will be safe.  My advice is to review the binlogs for a WinAppSDK app and do a before/after comparison to see what changed -- and I'd expect we'd want to change as little as possible.

Reply:
I thought the issue was with WinUI eagerly referencing winmds that didn't belong to it - is this a workaround for that?  Shouldn't we fix that properly?

Reply:
@Scottj1s I should have explained better.  Right now, the targets file in WinUI is pulling in *.winmd, and some of the winmds here are getting pulled in by that targets file.  I'm suggesting WinUI targets files only reference Microsoft.UI*.winmd, and this file speaks for the winmds in this repo.

Reply:
my apologies - wasn't paying close attention.  yes, I agree with that strategy - fewer and/or most narrowly scoped the wildcards, the better.